CONTRIBUTING.md

```diff
@@ -175,7 +175,9 @@ See the documentation wiki to learn more about [image partials][6].
 
 ### Links
 
-Avoid vague link text, let readers know where you're sending them. Any sentence containing a link should read just as well if it didn't contain the link.
+Format links using numbered [reference-style links][8], and use relative paths for other pages within the docs site. For example, instead of embedding the URL directly in the text, write `read the [Getting Started with Azure][1]` and define the link reference at the bottom of the file like `[1]: /getting_started/azure/`.
```

Comment:
```suggestion
Format links using numbered [reference-style links][8], and use relative paths for other pages within the documentation repository. For example, instead of embedding the URL directly in the text, write `read the [Getting Started with Azure][1]` and define the link reference at the bottom of the file like `[1]: /getting_started/azure/`.
```

Reply:
```suggestion
Format links using numbered [reference-style links][8], and use relative paths for other pages published on the documentation site. For example, instead of embedding the URL directly in the text, write `read the [Getting Started with Azure][1]` and define the link reference at the bottom of the file like `[1]: /getting_started/azure/`.
```